src/app/container/version-modal/version-modal.component.html

```diff
@@ -33,7 +33,7 @@ <h4 class="modal-title">{{TagEditorMode[mode]}} Version Tag</h4>
                      placeholder="e.g. develop" [disabled]="!(mode=='Add')" />
             </div>
           </div>
-          <div class="form-group form-group-sm">
+          <div class="form-group form-group-sm" *ngIf="tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
```

Comment:
Same deal, rather than hide, I'd probably prefer to make it non-editable. 
To make hosted entries "feel" more integrated with the rest of Dockstore and less of an oddball.